Podcast/AuthenticateUserEndpointRequest.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  AuthenticateUserEndpointRequest.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 12/13/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SwiftyJSON
+
+enum SignInType {
+    case Facebook
+    case Google
+
+    var signInURL: String {
```

Comment:
Just make this url. Then can use it as signInType.url. Looks cleaner

Reply:
done